workflow.rst

```diff
@@ -612,21 +612,24 @@ Then you can access this metadata in your controller as follows::
         ;
 
         // or
-        $title = $workflow->getMetadataStore()
-            ->getWorkflowMetadata()['title'] ?? false
+        $max_num_of_words = $workflow->getMetadataStore()
```

Comment:
For PHP variables we use `$camelCase`, so please update this and the other occurrence to `$maxNumOfWords`. Thanks! Note: we can use `max_num_of_words` as the PHP array key.

Reply:
PHP variable name changed. 
The PHP array key name comes from the configuration example (line 528).
Thanks!